fonz/cli.py

```diff
@@ -0,0 +1,43 @@
+import click
+from fonz.connection import Fonz
+
+
+@click.group()
+def cli():
+    pass
+
+
+@click.command()
+@click.argument('url', envvar='LOOKER_BASE_URL')
```

Comment:
> I'm thinking we should convert client ID and client secret to required options instead of positional arguments. Thoughts on that?

That works for me 👍 

>  I think it's great if we can support three options for config: CLI option/environment variable (ideal for CI runs), or YAML config file (ideal for local dev when you don't want to type everything in).

I agree with this. I think to do this we would actually have to change them to options, right? Otherwise, it will always expect the environment variables. We would have to put in logic that figures out which to use at run time. 

Would we need to set a hierarchy for the options? i.e. `CLI >  YAML > Environment Variable?`